pkgs/by-name/ze/zensical/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  rustPlatform,
+  cargo,
+  rustc,
+  python3Packages,
+  versionCheckHook,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "zensical";
+  version = "0.0.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "zensical";
+    repo = "zensical";
+    tag = version;
+    hash = "sha256-tEHY4O+odAhFvLentogJPUqT+xYVi8z0HXCml0A3Adw=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit src;
+    hash = "sha256-ljNoAaqXAVbnF/9RbXdWvJAac8sh0W3v53YtgiNQEng=";
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.maturinBuildHook
+    rustPlatform.cargoSetupHook
+    cargo
+    rustc
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    deepmerge
+    markdown
+    pygments
+    pymdown-extensions
+    pyyaml
+  ];
+
+  nativeCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+
+  meta = {
+    description = "A modern static site generator by the creators of Material for MkDocs ";
```

```suggestion
    description = "Modern static site generator by the creators of Material for MkDocs";
```
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes